tools/codegen/gen.py

```diff
@@ -1169,10 +1175,17 @@ def make_file_manager(install_dir: str) -> FileManager:
         'schema_registrations': list(mapMaybe(RegisterSchema(schema_selector), native_functions)),
     })
 
-    cpu_fm.write('Operators.cpp', lambda: {
-        'definitions': list(mapMaybe(ComputeOperators(
-            Target.DEFINITION), native_functions)),
-    })
+    def key_func(fn: NativeFunction) -> str:
+        return fn.func.name.unambiguous_name()
+
+    cpu_fm.write_sharded(
```

Comment:
It might be marginally useful to have an `OperatorsEverything.cpp` file that doesn't get built but doesn't force you to grep for the correct shard when you want to look at metadata about a particular operator. the sharding for autograd codegen does something similar with `VariableTypeEverything.cpp` here: https://github.com/pytorch/pytorch/pull/62063

Reply:
Take a look at the first PR in this stack. The logic for `FooEverything.cpp` is all there in `write_sharded` :)

Reply:
didn't notice that on the first pass, nice :)